database/src/main/scala/cromwell/database/slick/tables/DataAccessComponent.scala

```diff
@@ -28,6 +28,6 @@ class DataAccessComponent(val driver: JdbcProfile)
       metadata.schema ++
       workflowMetadataSummaries.schema ++
       workflowStore.schema ++
-      jobStore.schema ++
-      backendKVStore.schema
+      backendKVStore.schema ++
+      jobStore.schema
```

Comment:
Assuming this was an unintentional change, unless you really care about the order of the two schemas, then revert this diff.